Dockerfile-build

```diff
@@ -30,6 +26,9 @@ RUN set -eux; \
          exit 1; \
          ;; \
     esac; \
+    apt-get update \
+    && apt-get install -y --no-install-recommends unzip wget openssl \
+    && rm -rf /var/lib/apt/lists/*; \
     BASE_URL="https://public.dhe.ibm.com/ibmdl/export/pub/systems/cloud/runtimes/java/meta/"; \
```

Comment:
To further reduce the number of layers in the image. Group a couple of sessions in the file:
1. Group lines 32-48 into a group
2. Group lines 45-54 into a group
3. Group lines 55-57 into a group

Reply:
I'm not sure I quite understand, but from the original issue I believe we should be installing and removing wget/unzip in the same layer of the docker container so that it's not cached, which is why I combined the multiple steps.

Reply:
From the Tech Topic discussion, we want to reduce the number of layers created on the image.  Therefore, we'd like to combine steps together wherever it makes sense(similar to what you did for line 29-31 to group multiple steps together into a single step). Just group the 3 groups that I listed into 3 chunks of steps.